tests/stubbing_tests.cpp

```diff
@@ -166,6 +167,8 @@ struct BasicStubbing : tpunit::TestFixture {
         Mock<SomeInterface> mock;
         When(Method(mock, funcRefArgs)).ReturnAndSet(1, _2 <= 3, _1 <= 2);
         When(Method(mock, procRefArgs)).ReturnAndSet(_1 <= 4, _2 <= 5).ReturnAndSet( _2 <= 6, _1 <= 7).ReturnAndSet(_2 <= 8);
+        std::vector<std::string> v{"str"};
```

Comment:
Might be nicer to have separate test functions. Otherwise it's becoming quite complicated and harder to read/maintain.

Reply:
I'd do it if you think so. 

Unfortunately, the number of test cases in a TestFixture is [currently limited ](https://github.com/eranpeer/FakeIt/blob/a71cbd690e3c7b307e63ea3e359e53f57f9b4845/tests/tpunit%2B%2B.hpp#L240)and it should be manually increased. Which is annoying. So, I was reluctant to separate a test case to multiple test cases.

Reply:
Hmmm.
Looking up TPUnit seems latest versions might not have such restrictions.
https://github.com/tpounds/tpunitpp

Might not want to update too much in one Pull Request though.

Reply:
That's good to know. So I'll go and split this test. 

And yeah, I agree; specially since I won't reach the limit in this PR. We can make a separate PR for updating TPUnit.

Reply:
Done.